packages/react/plugins/webpack.ts

```diff
@@ -15,9 +15,9 @@ function getWebpackConfig(config: Configuration) {
     {
       test: /\.svg$/,
       oneOf: [
-        // If coming from JS/TS file, then transform into React component using SVGR.
+        // If coming from JS/TS or MDX file, then transform into React component using SVGR.
         {
-          issuer: /\.[jt]sx?$/,
+          issuer: /\.[jtm][sd]x?$/,
```

Comment:
```suggestion
          issuer: /\.(js|ts|md)x?$/,
```

`.jd` `.td` `.ms` `.jdx` `.tdx` `.msx` should be ignored.

Reply:
you're right: done